### PR TITLE
Refactor file structure and rename files [resolves #75]

### LINKED ISSUE #75: Refactor file structure and file names
- Separate Home and Dashboard components into separate folders
- No abbreviations

### PR DESCRIPTION
**Pull request by jason00111**
- separate homepage and dashboard
- fix linting errors

### CODE REVIEW

File: src/components/dashboard/Dashboard.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import Body from './Body'
+
+class Dashboard extends Component {
+  render() {
+
+    return (
+      <div className="dashboard-landing">
+        <Navigation />
+        <SecondaryNavigation />
+        <Body widgets={this.props.widgets} />
```
Comment (jamestewartjr) [inline src/components/dashboard/Dashboard.js]: Add spacing between curly braces

File: src/components/landing/navigation/Navigation.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+const Button = React.createClass({
+  getInitialState () {
+    return {
+      isOpen: false
+    }
+  },
+
+  toggleMenu (event) {
+    // event.stopPropagation()
+    this.setState({isOpen: !this.state.isOpen})
```
Comment (jamestewartjr) [inline src/components/landing/navigation/Navigation.js]: Add spacing between curly braces and parameters of function

File: src/components/landing/navigation/Navigation.js [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  },
+
+  onClose () {
+    this.setState({isOpen: false})
```
Comment (jamestewartjr) [inline src/components/landing/navigation/Navigation.js]: Add spacing between curly braces and parameters

File: src/components/landing/navigation/Navigation.js [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+  },
+
+  componentDidMount () {
+    document.body.addEventListener('click', this.onClose)
```
Comment (jamestewartjr) [inline src/components/landing/navigation/Navigation.js]: Add spacing between parans.

File: src/components/landing/navigation/Navigation.js [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+  },
+
+  componentWillUnmount () {
+    document.body.removeEventListener('click', this.onClose)
```
Comment (jamestewartjr) [inline src/components/landing/navigation/Navigation.js]: Add spacing between parans.

File: src/components/landing/navigation/Navigation.js [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+  },
+
+  render () {
+    return <div>
+      <a onClick={this.toggleMenu} href='#'>More
```
Comment (jamestewartjr) [inline src/components/landing/navigation/Navigation.js]: Add spacing between curly brackets.

File: src/components/landing/navigation/Navigation.js [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+          <div className='more-triangle' />
+      </a>
+      <Menu isOpen={this.state.isOpen} />
```
Comment (jamestewartjr) [inline src/components/landing/navigation/Navigation.js]: Add spacing between curly brackets.

File: src/components/dashboard/Dashboard.js [hunk 8]
```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+
+import Navigation from './navigation/Navigation'
+import SecondaryNavigation from './secondary_navigation/SecondaryNavigation'
+import Body from './Body'
+
+class Dashboard extends Component {
+  render() {
+
```
Comment (jrob8577) [inline src/components/dashboard/Dashboard.js]: Remove whitespace

File: src/components/landing/navigation/Navigation.js [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+
+const Button = React.createClass({
+  getInitialState () {
+    return {
+      isOpen: false
+    }
+  },
+
+  toggleMenu (event) {
+    // event.stopPropagation()
```
Comment (jrob8577) [inline src/components/landing/navigation/Navigation.js]: Remove comments

File: src/components/landing/navigation/Navigation.js [hunk 10]
```diff
@@ -0,0 +1,76 @@
...
+  },
+
+  componentWillUnmount () {
+    document.body.removeEventListener( 'click', this.onClose )
+  },
+
+  render () {
+    return <div>
+      <a onClick={ this.toggleMenu } href='#'>More
+          <div className='more-triangle' />
```
Comment (jrob8577) [inline src/components/landing/navigation/Navigation.js]: Indentation off

**Pull request merged**